sentry/src/main/java/io/sentry/protocol/SentryStackTrace.java

```diff
@@ -72,21 +79,29 @@ public SentryStackTrace(List<SentryStackFrame> frames) {
    *
    * @param frames the frames.
    */
-  public void setFrames(List<SentryStackFrame> frames) {
+  public void setFrames(final @Nullable List<SentryStackFrame> frames) {
     this.frames = frames;
   }
 
   @ApiStatus.Internal
   @Override
-  public void acceptUnknownProperties(Map<String, Object> unknown) {
+  public void acceptUnknownProperties(final @Nullable Map<String, Object> unknown) {
     this.unknown = unknown;
   }
 
-  public Map<String, String> getRegisters() {
+  public @Nullable Map<String, String> getRegisters() {
     return registers;
   }
 
-  public void setRegisters(Map<String, String> registers) {
+  public void setRegisters(final @Nullable Map<String, String> registers) {
     this.registers = registers;
   }
+
+  public @Nullable Boolean getSnapshot() {
```

Comment:
`m`: Java naming convention.
```suggestion
  public @Nullable Boolean isSnapshot() {
```

Reply:
the convention `is` is for the `boolean` primitive type, but here the object is nullable.